core/src/main/java/org/sonarsource/sonarlint/core/local/only/XodusLocalOnlyIssueStore.java

```diff
@@ -118,6 +120,63 @@ public void storeLocalOnlyIssue(String configurationScopeId, LocalOnlyIssue issu
     });
   }
 
+  public boolean reopenIssue(UUID issueId) {
```

Comment:
Here we are in the store so I would expect a more technical name, like remove. same for methods below